src/main/java/com/conveyal/r5/analyst/cluster/TravelTimeSurfaceRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.conveyal.r5.analyst.cluster;
+
+import com.conveyal.r5.profile.ProfileRequest;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Request a travel time surface containing travel times to all destinations for all iterations.
+ */
+public class TravelTimeSurfaceRequest extends GenericClusterRequest {
+    /** The profile request to use */
+    public ProfileRequest request;
+
+    public int zoom;
+    public int west;
+    public int north;
+    public int width;
+    public int height;
```

Comment:
It might be useful to allow the requester to specify the number of percentiles, considering that we usually only show 5 of them.